src/processes.c

```diff
@@ -844,73 +809,73 @@ static void ps_submit_proc_stats (
     dispatch_value_helper(&vl, "ps_code", NULL, 1, doing_detail, config->ps_code);
 
-    vl.values[0].gauge = stack_size;
+    vl.values[0].gauge = procstat_data.stack_size;
     dispatch_value_helper(&vl, "ps_stacksize", NULL, 1, doing_detail,
                           config->ps_stacksize);
 
-    vl.values[0].derive = vmem_minflt_counter;
-    vl.values[1].derive = vmem_majflt_counter;
+    vl.values[0].derive = procstat_data.vmem_minflt_counter;
+    vl.values[1].derive = procstat_data.vmem_majflt_counter;
     dispatch_value_helper(&vl, "ps_pagefaults", NULL, 2, doing_detail,
                           config->ps_pagefaults);
 
-    vl.values[0].derive = cpu_user_counter;
-    vl.values[1].derive = cpu_system_counter;
+    vl.values[0].derive = procstat_data.cpu_user_counter;
+    vl.values[1].derive = procstat_data.cpu_system_counter;
     dispatch_value_helper(&vl, "ps_cputime", NULL, 2, doing_detail,
                           config->ps_cputime);
 
-    if ( (io_rchar != -1) && (io_wchar != -1) )
+    if ( (procstat_data.io_rchar != -1) && (procstat_data.io_wchar != -1) )
     {
-        vl.values[0].derive = io_rchar;
-        vl.values[1].derive = io_wchar;
+        vl.values[0].derive = procstat_data.io_rchar;
+        vl.values[1].derive = procstat_data.io_wchar;
         dispatch_value_helper(&vl, "io_octets", NULL, 2, doing_detail,
                 config->ps_disk_octets);
     }
 
-    if ( (io_syscr != -1) && (io_syscw != -1) )
+    if ( (procstat_data.io_syscr != -1) && (procstat_data.io_syscw != -1) )
     {
-        vl.values[0].derive = io_syscr;
-        vl.values[1].derive = io_syscw;
+        vl.values[0].derive = procstat_data.io_syscr;
+        vl.values[1].derive = procstat_data.io_syscw;
         dispatch_value_helper(&vl, "io_ops", NULL, 2, doing_detail,
                               config->ps_disk_ops);
     }
 
-    if ( (io_diskr != -1) && (io_diskw != -1) )
+    if ( (procstat_data.io_diskr != -1) && (procstat_data.io_diskw != -1) )
     {
-        vl.values[0].derive = io_diskr;
-        vl.values[1].derive = io_diskw;
+        vl.values[0].derive = procstat_data.io_diskr;
+        vl.values[1].derive = procstat_data.io_diskw;
         dispatch_value_helper(&vl, "disk_octets", NULL, 2, doing_detail,
                               config->ps_disk_octets);
     }
 
     if ( report_ctx_switch )
     {
-	    vl.values[0].derive = cswitch_vol;
+	    vl.values[0].derive = procstat_data.cswitch_vol;
 	    dispatch_value_helper(&vl, "contextswitch", "voluntary", 2, doing_detail,
 		    config->cswitch_vol);
-	    vl.values[0].derive = cswitch_invol;
+	    vl.values[0].derive = procstat_data.cswitch_invol;
 	    dispatch_value_helper(&vl, "contextswitch", "involuntary", 2, doing_detail,
 		    config->cswitch_invol);
     }
     meta_data_destroy(vl.meta);
     vl.meta = NULL;
 
-    DEBUG ("name = %s; num_proc = %lu; num_lwp = %lu; "
+    DEBUG ("name = %s; pid = %s; num_proc = %lu; num_lwp = %lu; "
```

Comment:
This change isn't strictly part of this PR, is it? It's ok to keep it, but let's update the description.

Reply:
Done.

Reply:
Where?

Reply:
I updated the description in the PR at the bottom. I'm not sure what other description you're referring to.

Reply:
Ok.